doc/api/child_process.md

```diff
@@ -1495,8 +1496,9 @@ added: REPLACEME
 
 Child processes support a serialization mechanism for IPC that is based on the
 [serialization API of the `v8` module][v8.serdes]. This is generally more
-powerful and supports more built-in JS object types, such as `BigInt`, `Map`
-and `Set`, `ArrayBuffer` and `TypedArray`, `Buffer`, `Error`, `RegExp` etc.
+powerful and supports more built-in JavaScript object types, such as `BigInt`,
+`Map` and `Set`, `ArrayBuffer` and `TypedArray`, `Buffer`, `Error`,
```

Comment:
instead of listing these you could link to the mdn page about structured cloning

Reply:
I’ve added a link but I still feel like the list here is more helpful than just a link. (And we also had the link from the v8.md serdes section too, this just removes one click.)